cmd/cmd.go

```diff
@@ -122,3 +125,30 @@ func currentPath() (string, bool) {
 	}
 	return path.Dir(filename), true
 }
+
+const PreferredNumOpenFiles = 10000
```

Comment:
* Need a comment here to make golint happy.
* Have you experimented with this number at all? Any chance we can get away with, say 5000?

Reply:
Erm, seems like others use the 10000 figure, which may be arbitrary, but we don't have time to tinker with _everything_ 😄, maybe we can revisit this const later on